lib/pdk/cli/bundle.rb

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
I think this blank line shouldn't be here; I'm surprised rubocop doesn't complain about it